src/components/Company/CompanyName/CorrectCompanyName.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <v-form ref="correctNameForm" v-model="valid" lazy-validation>
+    <v-layout row>
+      <v-flex>
+        <v-text-field
+          v-model="companyName"
+          filled
+          req
```

Comment:
Is this redundant, since you aleady have a rule to make this required?

Reply:
It is! Will fix in next commit